.gitignore

```diff
@@ -56,6 +56,8 @@ rspec.xml
 
 # Ignore certificates directory
 config/certs
+config/ca-trust
+!config/ca-trust/README.md
```

Comment:
Can i get a tl;dr of what ! does in gitignore? The documentation I found is a bit confusing.

We'd only want to ignore the `.pem` & `.crt` files, yes? In order to maintain a clean `git` state when developing on a review instance.

Reply:
I'm telling git to ignore that directory but exclude the README.md from that ignore ( gotta love a double-negative 😄 ). I'm ignoring everything there except that one file.

Yeah, I guess that would be another way to approach it. Either way is fine by me.